tests/rptest/tests/rbac_test.py

```diff
@@ -188,3 +188,19 @@ def expect_roles_with_filter(expected_roles: List[str],
         expect_roles_with_filter(filter="ccc", expected_roles=[])
 
         # TODO: add tests for user membership once assigning users to roles gets implemented
+
+    @cluster(num_nodes=3)
+    def test_get_role(self):
+        with expect_role_error(RoleErrorCode.ROLE_NOT_FOUND):
+            self.superuser_admin.get_role(role=self.role_name0)
+
+        self._create_and_wait_for_role(role=self.role_name1)
+
+        res = self.superuser_admin.get_role(role=self.role_name1)
+        res_json = res.json()
+        role_name = res_json['name']
+        role_members = res_json.get('members', [])
+        assert role_name == self.role_name1, f"Unexpected role name: {role_name} != {self.role_name1}"
+        assert role_members == [], f"Unexpected role members: {role_members} != {[]}"
```

Comment:
Nice! :) Should we do this as a follow up though to avoid conflicts? I left a few TODOs anyway to assert role membership changes in a few tests, so we could address this along with them either as a follow-up PR or even here if we wait for your PR to get merged first

Reply:
Yep, nothing to do here